### PR TITLE
fail build if app is using NoN already

### PR DESCRIPTION
**Pull request by lindsaylevine**
Fixes https://github.com/netlify/netlify-plugin-nextjs/issues/47

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,12 +37,16 @@ module.exports = {
       return failBuild(`** Static HTML export next.js projects do not require this plugin **`)
     }
 
-    // TO-DO: check scripts to make sure the app isn't manually running NoN
-    // For now, we'll make it clear in the README
-    // const isAlreadyUsingNextOnNetlify = Object.keys(dependencies).find((dep) => dep === 'next-on-netlify');
-    // if (isAlreadyUsingNextOnNetlify) {
-    //   return failBuild(`This plugin cannot support apps that manually use next-on-netlify. Uninstall next-on-netlify as a dependency to resolve.`);
-    // }
+    const hasNextOnNetlifyInstalled = Object.keys(dependencies).find((dep) => dep === 'next-on-netlify')
```
Comment (ehmicky) [inline index.js:40]:
```suggestion
    const hasNextOnNetlifyInstalled = dependencies['next-on-netlify'] !== undefined
```

File: index.js [hunk 2]
```diff
@@ -37,12 +37,16 @@ module.exports = {
...
+    const hasNextOnNetlifyPostbuildScript = Object.keys(scripts).find(
+      (script) => script === 'postbuild' && scripts[script].includes('next-on-netlify'),
+    )
```
Comment (ehmicky) [inline index.js:43]: Would the following be simpler?

```js
    const hasNextOnNetlifyPostbuildScript = typeof scripts.postbuild === 'string' && scripts.postbuild.includes('next-on-netlify')
```

File: index.js [hunk 3]
```diff
@@ -37,12 +37,16 @@ module.exports = {
...
+    const isAlreadyUsingNextOnNetlify = hasNextOnNetlifyInstalled || hasNextOnNetlifyPostbuildScript
+    if (isAlreadyUsingNextOnNetlify) {
+      return failBuild(
+        `This plugin does not support apps that manually use next-on-netlify. Uninstall next-on-netlify as a dependency to resolve.`,
```
Comment (ehmicky) [inline index.js:47]:
```suggestion
        `This plugin does not support sites that manually use next-on-netlify. Uninstall next-on-netlify as a dependency to resolve.`,
```

**Review by ehmicky — COMMENTED** [review]
Sweet! :tada: 
Some minor comments.

File: index.js (hunk 2 above)
Comment (lindsaylevine) [reply index.js:43]: yes and more reliable, idk why i clung to Object.keys and somehow forgot object values can be accessed at object[key] lul ty

**Review by ehmicky — APPROVED** [review]

**Pull request merged**